.github/workflows/sourcecode-release.yaml

```diff
@@ -13,9 +13,26 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
+
+      - name: Prepare release notes
+        run: |
+          touch release-notes.md
+          echo "### Images and binaries here: https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/${{ github.ref_name }}" >> release-notes.md
+          echo "" >> release-notes.md
+          echo "## End User Changelog" >> release-notes.md
+
+          awk '/<!-- next version -->/,/<!-- previous-version -->/' CHANGELOG.md > tmp-chlog.md # select changelog of latest version only
+          sed '1,3d' tmp-chlog.md >> release-notes.md # delete first 3 lines of file
```

Comment:
Based on the output I see that this works, but out of curiosity, what's on the first three lines that we need to delete?

Reply:
it's some newlines and the `## <version>` heading that we don't need for the release